### ISSUE TITLE
No change in accuracy even with 200 samples

### ISSUE THREAD

**Issue by cogmeta** [friday27/KeywordSpotting#2]
('Epoch:', 10)
Training   Accuracy = 48.438 %   Training   Loss = 0.693659
Validation Accuracy = 43.750 %   Validation Loss = 0.696315
('Checkpoint:', 'ckpt/2018-04-21/kws_v1-20180421-1025-10')
============================================================
('Epoch:', 11)
Training   Accuracy = 48.438 %   Training   Loss = 0.693522
Validation Accuracy = 43.750 %   Validation Loss = 0.696481
('Checkpoint:', 'ckpt/2018-04-21/kws_v1-20180421-1025-11')
============================================================
('Epoch:', 12)
Training   Accuracy = 48.438 %   Training   Loss = 0.693392
Validation Accuracy = 43.750 %   Validation Loss = 0.696657
('Checkpoint:', 'ckpt/2018-04-21/kws_v1-20180421-1025-12')
============================================================
('Epoch:', 13)
Training   Accuracy = 48.438 %   Training   Loss = 0.693266
Validation Accuracy = 43.750 %   Validation Loss = 0.696841
('Checkpoint:', 'ckpt/2018-04-21/kws_v1-20180421-1026-13')
============================================================
('Epoch:', 14)
Training   Accuracy = 48.438 %   Training   Loss = 0.693142
Validation Accuracy = 43.750 %   Validation Loss = 0.697036
('Checkpoint:', 'ckpt/2018-04-21/kws_v1-20180421-1026-14')
